.github/workflows/pull-cli-docs.yaml

```diff
@@ -0,0 +1,21 @@
+name: pull-cli-docs
+on:
+  pull_request:
+    branches:
+      - main
+      - 'release-**'
+  workflow_dispatch:
+jobs:
+  validate-docs:
+    name: docs validation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Kyma CLI
+        uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+          cache-dependency-path: 'go.sum'
+      - name: Run Docs Validation
+        run: make validate
```

Comment:
Does this include linting?
We should also use the golangci-lint action please in a separate step within this job!
Code linting
Docs linting

Reply:
The code linting is done in an another Job